{{cookiecutter.project_name}}/.mlops-setup-scripts/README.md

```diff
@@ -198,6 +200,9 @@ python .mlops-setup-scripts/cicd/bootstrap.py \
 {%- if cookiecutter.cicd_platform == "gitHub" %}
   --var github_repo_url=https://github.com/<your-org>/<your-repo-name> \
   --var git_token=<your-git-token>
+{%- elif cookiecutter.cicd_platform == "gitHubEnterprise" %}
+   -- var github_repo_url=<your-enterprise-github-server>/<your-org>/<your-repo-name> \
+   -- var git_token=<your-git-token>
```

Comment:
```suggestion
   --var github_repo_url=<your-enterprise-github-server>/<your-org>/<your-repo-name> \
   --var git_token=<your-git-token>
```
Also would it be `github_repo_url` or `github_enterprise_repo_url`?

Reply:
I use the same variable for `gitHub` and `gitHubEnterprise` to reduce complexity since we only allow one choice.